kinode/packages/homepage/ui/src/index.css

```diff
@@ -1,226 +1,392 @@
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 10px;
+  margin-bottom: 0;
+  background-color: transparent !important;
+  padding: 0em !important;
 }
 
 .widget {
-    border: 1px solid black;
-    border-radius: 10px;
-    text-align: center;
-    position: relative;
-    max-width: 100%;
-    width: 300px;
-    height: 400px;
-    overflow: hidden;
+  color: var(--tasteful-dark);
+  background-color: var(--off-white);
+  border: 1px solid var(--tasteful-dark);
+  /* border-radius: 10px; */
+  text-align: center;
+  position: relative;
+  max-width: 100vw;
+  width: 338px;
+  height: 400px;
+  overflow: hidden;
 }
 
 .widget iframe {
-    border: none;
-    width: 100%;
-    height: 100%;
+  border: none;
+  width: 100%;
+  height: 100%;
 }
 
 .widget .bottom-bar {
-    display: none;
-    position: absolute;
-    bottom: 0;
-    border-top: 1px solid black;
-    background-color: var(--blue);
-    width: 100%;
-    padding: 2px;
-    flex-direction: row;
-    justify-content: space-between;
+  display: none;
+  position: absolute;
+  bottom: 0;
+  border-top: 1px solid black;
+  background-color: var(--blue);
+  width: 100%;
+  padding: 2px;
+  flex-direction: row;
+  justify-content: space-between;
+  border-color: var(--off-white);
 }
 
 .widget:hover .bottom-bar {
-    display: flex;
+  display: flex;
 }
 
 .widget .bottom-bar p {
-    font-size: 0.8em;
-    cursor: default;
+  font-size: 0.8em;
+  cursor: default;
 }
 
 footer {
-    text-align: center;
-    position: fixed;
-    bottom: 0;
-    width: 90vw;
-    max-height: 100vh;
-    max-width: 960px;
-    left: 50%;
-    transform: translate(-50%, 0);
+  text-align: center;
+  max-height: 100vh;
+  max-width: 100vw;
 }
 
-.footer-button {
-    background-color: var(--blue);
-    color: light-dark(var(--off-black), var(--off-white));
-    border-radius: 25px 25px 0px 0px;
-}
 
 #all-apps {
-    border: 1px solid black;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    gap: 10px;
-    justify-content: space-evenly;
-    padding: 10px;
-    background-color: light-dark(var(--gray), var(--tasteful-dark));
-    border-radius: 25px 25px 0px 0px;
+  background-color: var(--off-black);
+  border-color: var(--off-white);
+  border-radius: 0 0 1em 1em;
 }
 
-.app-icon {
-    width: 64px;
-    height: 64px;
-    margin: 0 auto;
+.apps-grid {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  width: 100%;
+  color: var(--off-white);
 }
 
-.modal {
+.apps-grid.apps-count-3 {
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.expand-button {
+  width: 100%;
+  background-color: transparent;
+  border-radius: 0 0 1em 1em;
+  border: 0.5px solid rgba(255, 255, 255, 0.2);
+  padding: 1em;
+  color: var(--white);
+}
+
+@media (max-width: 1024px) {
+  .apps-grid {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (max-width: 768px) {
+  body,
+  html {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+  }
```

Comment:
How much of this (and other stuff in this file) makes sense to expose globally in kinode.css?

Reply:
imo assuming a body structure and things like mobile width adjustments might be too opinionated if people want to pull in kinode.css into their UIs